### DISCUSSION TITLE
Container's App Icons System Tray Integration (All Desktops) [Suggestion]

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [89luca89/distrobox#1903] [Ideas]
**Is your feature request related to a problem? Please describe.**
I adapted the docker container solution for criptext from [here](https://github.com/Criptext/Criptext-Email-React-Client/issues/1531) to run via distrobox instead, which was a success. Except it's missing the system tray icon
**Describe the solution you'd like**
Somehow auto-integrate apps that want to have icons in the system tray so they show up in the host system tray on any DE (I'm using KDE Plasma, for example). This should also work for exported apps & binaries...
**Describe alternatives you've considered**
I don't think there are any, or ever will be because of criptext being practically abandonware at this point...
**Additional context**

**Comment by ghost** [comment]
Would you plan to add this? I have the same problem.

**Comment by dottorblaster** [comment]
I don't know if it's even possible. Moving to discussions to keep brainstorming on this. Any help is appreciated.
